### DISCUSSION TITLE
change user mail address or reset password in admin portal

### DISCUSSION THREAD

**Discussion by burghy86** [dani-garcia/vaultwarden#4991] [Q&A]
### Vaultwarden Support String

Server Installed Ok
    1.32.0 
Server Latest
    1.32.0 
Web Installed
    2024.6.2b 
Database
    SQLite: 3.46.0 

Checks

OS/Arch
    linux / x86_64 
Running within a container
    Yes (Base: Debian) 
Environment settings overridden
    Yes 
Uses a reverse proxy
    Yes 
IP header Match
    Config/Server: X-Real-IP 
Internet access Ok
    Yes 
Internet access via a proxy
    No 
DNS (github.com) Ok
    140.82.121.4 
Date & Time (Local)
    Server: 2024-09-25 10:46:10 +00:00 
Date & Time (UTC) Server/Browser Ok Server NTP Ok Browser NTP Ok
    NTP: 2024-09-25 10:46:10 UTC Server: 2024-09-25 10:46:10 UTC Browser: 2024-09-25 10:46:16 UTC 

### Vaultwarden Build Version

1.32.0 

### Deployment method

Official Container Image

### Custom deployment method

I have a Vaultwarden instance with 10 registered users. One of the users signed up using an email address from their old company. As an admin, I can't find a way to change the user's email address. If the user has forgotten their password, they won't be able to reset their account, as the reset email would go to the old, inaccessible address.

Is there a way for an admin to update a user's email? If not, are there any recommended solutions to resolve this issue?

Thank you for your help!

### Reverse Proxy

nginix

### Host/Server Operating System

Linux

### Operating System Version

synology

### Clients

Web Vault

### Client Version

chrome

### Steps To Reproduce

in user config i can only delete o disable user but not reset password o change mail

### Expected Result

.

### Actual Result

.

**Comment by stefan0xC** [comment]
This is not possible as far as I know because [it's used as salt to derive the master key](https://bitwarden.com/help/bitwarden-security-white-paper/#account-creation).

The user can change their own email address without any access to the old email address required. And if you want to be able to change your users passwords you should let them join an organization [with the account recovery policy automatically enabled](https://bitwarden.com/help/policies/#account-recovery-administration).

**Comment by BlackDex** [comment]
Also, a password reset doesn't exist. It will destroy the account and it will create a new one. You can't reset your own account with the master password.

Also, an other option would be te fetch that mail on your mail server. But in the end it will be deleted anyway, so then you can just delete the account from the admin interface anyway.
